qiita_pet/templates/list_studies.html

```diff
@@ -0,0 +1,266 @@
+{% extends sitebase.html %}
+{% block head %}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
+
+<script type="text/javascript">
+var current_study;
+var ajaxURL = "/study/search/?&user={{current_user.id}}&sEcho=" + Math.floor(Math.random()*1001);
```

Comment:
The documentation on this parameter is extremely sparse. Most users' experiences are that it must be specified, but that the value seems not to matter very much. [This is the official word](http://legacy.datatables.net/usage/server-side) on it, and it does mention XSS attacks, but how it helps, I don't know.